### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by MatthijsSmets**
Angular.json was updated in [ladybug-frontend](https://github.com/wearefrank/ladybug-frontend) which requires an update to the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,38 +155,34 @@ properties as describes in the comments in this file.
...
-* Use `yarn -v` to check that you have yarn version 3.2.x.
-* Run `yarn install --frozen-lockfile` to install your dependencies. The flag ensures that you get exactly the same dependencies as the other developers have. If you want to include a new dependency, update `package.json` and do `yarn install`. File `yarn.lock` will be updated, the list of all dependencies including dependencies of other dependencies. Check in `yarn.lock` to ensure that other developers will update to the same dependencies as you. In case you don't have a direct internet connection you might need to set HTTPS_PROXY, see https://docs.cypress.io/guides/references/proxy-configuration.
-* Run `yarn prepare` to prepare Git hooks. If you do not do this, you will not be able to commit or push in the ladybug-frontend project.
 
-Testing frontend changes with the test webapp
----------------------------------------------
+- Install [Node.js](https://nodejs.org/en/), choose version 16. You should get executables `npm` version 8.x and `node` version 16.x. Check these versions using `npm -v` and `node -v`.
```
Comment (philipsens) [inline README.md:169]: Is there a reason to change the syntax of the list items?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
-Ladybug debug- and test tool by WeAreFrank!
-===========================================
+# Ladybug debug- and test tool by WeAreFrank!
```
[the remark this answers is not in the document]
Comment (MatthijsSmets) [reply README.md:1]: I did not change this, let me check how this happened

**Review by philipsens — CHANGES_REQUESTED** [review]
It's hard to review the changes will all the changed syntax, and indent. 

I'd suggest to put it back.

File: README.md [hunk 3]
```diff
@@ -240,12 +240,8 @@ Create and publish NPM package and WebJar
...
 
 ### Creating and publishing an NPM package
-- Run the command `yarn install --frozen-lockfile`, to sync with changes done by others
+- Run the command `yarn install --immutable`, to sync with changes done by others
 - Run the command `ng build`, to build the current project
-- Copy the following files into the generated `dist/ladybug` folder
-    - README.md
-    - LICENSE
-    - package.json
 - In the `dist/ladybug/index.html` change the types of the three scripts on line 15 to `application/javascript` (so `type="module"` -> `type="application/javascript"`). The specific scripts are:
```
Comment (mhdirkse) [inline README.md:245]: On my computer it turned out to be line 14.

**Review by mhdirkse — CHANGES_REQUESTED** [review]
You need `npm publish -otp=<code from authenticator app of your telephone>`

**Comment by MatthijsSmets** [comment]
> You need `npm publish -otp=<code from authenticator app of your telephone>`

This is not the case for every user, it depends on if you have 2FA enabled.

File: README.md [hunk 4]
```diff
@@ -240,12 +240,8 @@ Create and publish NPM package and WebJar
     - In the terminal run the following command `npm login`, which prompts you to log into NPM.
 
 ### Creating and publishing an NPM package
-- Run the command `yarn install --frozen-lockfile`, to sync with changes done by others
+- Run the command `yarn install --immutable`, to sync with changes done by others
 - Run the command `ng build`, to build the current project
```
[the remark this answers is not in the document]
Comment (MatthijsSmets) [reply README.md:244]: That means your angular cli is not installed properly, this still works for anyone that has it installed correctly.
This is also something that was not changed in this pr.

**Review by philipsens — APPROVED** [review]

**Review by jacodg — APPROVED** [review]

**Pull request merged**